src/domains/resource/application/allocation.result.resource.finder.ts

```diff
@@ -0,0 +1,66 @@
+import { NotificationResourceFinder } from './notification.resource.finder';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { AllocationRepository } from '../../allocation/domain/allocation.repository';
+import { AllocationResultResource } from '../domain/allocation.result.resource';
+import { AllocationEntity } from '../../allocation/domain/allocation.entity';
+import { CarEntity } from '../../car/domain/car.entity';
+import { CallRequestEntity } from '../../call_request/domain/call.request.entity';
+import { ServiceUserEntity } from '../../service_user/domain/service.user.entity';
+import { CarModelEntity } from '../../car_model/domain/car.model.entity';
+import { CharterRepository } from '../../charter/domain/charter.repository';
+import { CharterEntity } from '../../charter/domain/charter.entity';
+
+@Injectable()
+export class AllocationResultResourceFinder
+  implements NotificationResourceFinder
+{
+  constructor(
+    private readonly allocationRepository: AllocationRepository,
+    private readonly charterRepository: CharterRepository,
+  ) {}
+
+  /**
+   * 배차 ID를 받아 배차 엔티티를 포함하여 배차 결과 알림 메시지에 필요한 리소스들을 구한다.
+   * 배차, 대절 엔티티는 레포지터리를 이용하여 구하고, 나머지는 배차를 통해 조회한다.
+   * @param allocId - 배차 ID
+   * @returns: allocationResultResource - 배차 결과 관련 리소스
+   */
+  async find(allocId: number): Promise<AllocationResultResource> {
+    const allocation = await this.allocationRepository.findByAllocId(allocId);
+    if (!allocation) {
+      throw new NotFoundException('배차 ID 오류');
+    }
```

Comment:
```suggestion
    if (!allocation) {
      throw new NotFoundException('배차 아이디가 유효하지 않습니다.(${allocId})');
    }
```
- 에러에서는 명시적으로 에러가 발생한 원인이 되는 정보들을 추가적으로 적어주시면 좋습니다.